### PR TITLE
Customizing the Okta-hosted sign-in widget and adding a custom domain

### PR DESCRIPTION
**Pull request by alisaduncan**
This blog post has the following features:

- [ ] A GitHub Repository with a polished README
- [ ] A GitHub Repository under the github.com/oktadev account
- [x] A title that's approved by Dev Advocacy
- [x] A URL that's approved by Dev Advocacy
- [x] The content has been run through Grammarly (https://www.grammarly.com/)
- [x] Rendered locally and confirmed that no Markdown typos exist

### CODE REVIEW

**Comment by alisaduncan** [comment]
[Content - adding a custom domain and spiffing up the Okta hosted](https://trello.com/c/o53xQds3/5-content-adding-a-custom-domain-and-spiffing-up-the-okta-hosted)

File: _source/_posts/2023-01-04-signin-custom-domain.md [hunk 1]
```diff
@@ -0,0 +1,123 @@
...
+- "Add some flair to your sign-in process by following the quick steps in this post!"
+image: blog/signin-custom-domain/social.jpg
+type: awareness
+---
+
+Creating secure applications requires authentication. Delegating all the tedious details of the sign-in process to Okta is the most secure method to authenticate, not to mention speedier for development. So you'll see us advocating for and using the Okta-hosted sign-in page in our blog posts.
+
+But the default sign-in page can look too different from the personality you have in your application. And the login URL redirects to a domain outside your application, which is also different from what we want. Fortunately, you can add some flair to your Okta-hosted sign-in page, allowing your application's character to carry through in the user authentication process while maintaining the highest level of security. In other words, winning with the best of both worlds.
+
+![Giphy of Awkafina holding a trophy with confetti falling around her](https://media.giphy.com/media/26u4cqiYI30juCOGY/giphy.gif)
```
Comment (bdemers) [inline _source/_posts/2023-01-04-signin-custom-domain.md]: nit: center image, not a big deal though, so don't worry about it too much, but _i think_ something like this _should_ work:
```suggestion
![Giphy of Awkafina holding a trophy with confetti falling around her](https://media.giphy.com/media/26u4cqiYI30juCOGY/giphy.gif){: .center-image}
```

File: _source/_posts/2023-01-04-signin-custom-domain.md [hunk 2]
```diff
@@ -0,0 +1,123 @@
...
+
+> **Note**
+The process outlined in this post is not Angular-dependent, so feel free to substitute the web app with your favorite SPA application.
+
+{% img blog/signin-custom-domain/themed-sign-in.gif alt:"Animated gif of sign-in from a cupcake site" width:"800" %}{: .center-image }
+
+**Prerequisites**
+- [Node](https://nodejs.org/)
+- [Okta CLI](https://cli.okta.com)
+- ***Optional** Domain name you own and access to administrate DNS of the domain
```
Comment (bdemers) [inline _source/_posts/2023-01-04-signin-custom-domain.md]:
```suggestion
- **Optional** Domain name you own and access to administrate DNS of the domain
```

**Review by bdemers — APPROVED** [review]
I ❤️ 🧁

A couple of nits, the `*Optional` one might be intentional, if so, ignore my comments 😉

File: _source/_posts/2023-01-04-signin-custom-domain.md (hunk 1 above)
Comment (alisaduncan) [reply _source/_posts/2023-01-04-signin-custom-domain.md]: Yup! This works.

File: _source/_posts/2023-01-04-signin-custom-domain.md (hunk 2 above)
Comment (alisaduncan) [reply _source/_posts/2023-01-04-signin-custom-domain.md]: Yup, this one was intentional. :)

**Review by bdemers — APPROVED** [review]

**Pull request merged**